torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2552,17 +2557,66 @@ def is_strict_for(v: VariableTracker):
         with tx.output.subtracer(fwd_fn, fwd_tracer), tx.strict_translation_mode(
             is_strict_for
         ):
-            (bwd_out, _), bwd_graph, bwd_freevars = speculate_subgraph(
-                tx,
-                bwd_fn,
-                bwd_args,
-                kwargs,
-                "autograd.Function",
-                enable_grad=False,
-                set_subgraph_inputs="manual",
-                restore_side_effects=False,
-                tracer=bwd_tracer,
-            )
+            try:
+                (bwd_out, _), bwd_graph, bwd_freevars = speculate_subgraph(
+                    tx,
+                    bwd_fn,
+                    bwd_args,
+                    kwargs,
+                    "autograd.Function",
+                    enable_grad=False,
+                    set_subgraph_inputs="manual",
+                    restore_side_effects=False,
+                    tracer=bwd_tracer,
+                )
+            except torch._dynamo.exc.Unsupported as e:
+                from unittest import mock
+
+                if e.msg == "Illegal getattr invocation is_contiguous in strict mode":
```

Comment:
Can you make the Illegalgetattrinvocation a specific exception type that we can handle in an except block here?

Also, it's not just is_contiguous, we can also handle .stride() calls this way.